app/templates/core/create-account-modal/basic-info-view.jade

```diff
@@ -0,0 +1,105 @@
+form#basic-info-form.modal-body.basic-info
+  .modal-body-content
+    .auth-network-logins.text-center
+      h4
+        span(data-i18n="signup.connect_with")
+      a#facebook-signup-btn.btn.btn-primary.btn-lg.btn-illustrated.network-login(disabled=!view.signupState.get('facebookEnabled'), data-sso-used="facebook")
+        img.network-logo(src="/images/pages/modal/auth/facebook_sso_button.png", draggable="false", width="175", height="40")
+        span.sign-in-blurb(data-i18n="login.sign_in_with_facebook")
+      a#gplus-signup-btn.btn.btn-danger.btn-lg.btn-illustrated.network-login(disabled=!view.signupState.get('gplusEnabled'), data-sso-used="gplus")
+        img.network-logo(src="/images/pages/modal/auth/gplus_sso_button.png", draggable="false", width="175", height="40")
+        span.sign-in-blurb(data-i18n="login.sign_in_with_gplus")
+        .gplus-login-wrapper
+          .gplus-login-button
+    
+    .hr-text
+      hr
+      span(data-i18n="general.or")
+                  
+    div.form-container
+      if ['student', 'teacher'].indexOf(view.signupState.get('path')) !== -1
+        .row.full-name
+          .col-xs-offset-3.col-xs-5
+            .form-group
+              label.control-label(for="first-name-input")
+                span(data-i18n="general.first_name")
+              input#first-name-input.form-control.input-lg(name="firstName")
+          .col-xs-4
+            .last-initial.form-group
+              label.control-label(for="last-name-input")
+                span(data-i18n="general.last_initial")
+              input#last-name-input.form-control.input-lg(name="lastName" maxlength="1")
+      .form-group
+        .row
+          .col-xs-5.col-xs-offset-3
+            label.control-label(for="email-input")
+              span(data-i18n="share_progress_modal.form_label")
+          .col-xs-5.col-xs-offset-3
+            input.form-control.input-lg#email-input(name="email" type="email")
+          .col-xs-4.email-check
+            - var checkEmailState = view.state.get('checkEmailState');
+            if checkEmailState === 'checking'
+              span.small(data-i18n="signup.checking")
+            if checkEmailState === 'exists'
+              span.small
+                span.text-burgandy.glyphicon.glyphicon-remove-circle
+                =" "
```

Comment:
This is [one of the options they provided in an issue on the subject](https://github.com/pugjs/pug/issues/905#issuecomment-69221807). I think I prefer it to `.spr` and `.spl` (which is also kinda their Option 2) since, as you pointed out, that messes with copy and paste, and really it's kinda janky. Also prefer this method to their other suggested options, except maybe tag interpolation or HTML (options 5, 6) in some cases.